docs/reference/access-log-policy.md

```diff
@@ -0,0 +1,106 @@
+|---------------------------------------------|------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `destinationArn`                            | *string*                                                                                       | The ARN of the Amazon S3 Bucket, Amazon CloudWatch Log Group, or Amazon Kinesis Data Firehose Delivery Stream that will have access logs published to it. |
+| `targetRef`                                 | *[PolicyTargetReference](https://gateway-api.sigs.k8s.io/geps/gep-713/#policy-targetref-api)* | TargetRef points to the kubernetes `Gateway`, `HTTPRoute`, or `GRPCRoute` resource that will have this policy attached. This field is following the guidelines of Kubernetes Gateway API policy attachment. |
+
+## Example Configurations
+
+### Example 1
+
+This configuration results in access logs being published to the S3 Bucket, `my-bucket`, when traffic
+is sent to any HTTPRoute or GRPCRoute that is a child of Gateway `my-hotel`.
+
+```yaml
+apiVersion: application-networking.k8s.aws/v1alpha1
+kind: AccessLogPolicy
+metadata:
+  name: my-access-log-policy
+spec:
+  destinationArn: "arn:aws:s3:::my-bucket"
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: my-hotel
+```
+
+### Example 2
+
+This configuration results in access logs being published to the CloudWatch Log Group, `myloggroup`, when traffic
+is sent to HTTPRoute `inventory`.
+
+```yaml
+apiVersion: application-networking.k8s.aws/v1alpha1
+kind: AccessLogPolicy
+metadata:
+  name: my-access-log-policy
+spec:
+  destinationArn: "arn:aws:logs:us-west-2:123456789012:log-group:myloggroup:*"
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: inventory
+```
+
+## Statuses
+
+ALPs fit under the definition of [Gateway API Policy Objects](https://gateway-api.sigs.k8s.io/geps/gep-713/#on-policy-objects).
+As a result, status conditions are applied on every modification of an ALP, and can be viewed by describing the ALP.
+
+### Accepted
+
+The spec of the ALP is valid and has been successfully processed.
+
+### Conflicted
+
+The target already has an ALP for the same destination type
+(i.e. a target can have 1 ALP for an S3 Bucket, 1 ALP for a CloudWatch Log Group,
+and 1 ALP for a Firehose Delivery Stream at a time).
+
+### Invalid
+
+Any of the following reasons:
+- The target's `Group` is not `gateway.networking.k8s.io`
+- The target's `Kind` is not `Gateway`, `HTTPRoute`, or `GRPCRoute`
+- The target's namespace does not match the ALP's namespace
+- The destination with the given `destinationArn` could not be found
+- The controller is missing AWS permissions
```

Comment:
would it be worth mentioning what kind of AWS permissions are required to create this policy?

Reply:
Good point, will add this